policy/modules/services/xserver.if

```diff
@@ -335,6 +335,29 @@ interface(`xserver_non_drawing_client',`
 ## </param>
 #
 template(`xserver_common_x_domain_template',`
+	refpolicywarn(`$0($*) has been deprecated, use xserver_common_x_domain() instead.')
+	xserver_common_x_domain($*)
+')
+
+#######################################
+## <summary>
+##	Interface to provide X object permissions on a given X server to
+##	an X client domain.  Provides the minimal set required by a basic
+##	X client application.
+## </summary>
+## <param name="prefix">
+##	<summary>
+##	The prefix of the X client domain (e.g., user
+##	is the prefix for user_t).
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	Client domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`xserver_common_x_domain',`
```

Comment:
While this is not declaring any types, it is still a template because it uses `$1_input_xevent_t`; it is referencing a templated type.